libraries/types/pbft_block/include/pbft/pbft_block.hpp

```diff
@@ -129,6 +129,12 @@ class PbftBlock {
    * @brief Set PBFT block hash and block proposer address
    */
   void calculateHash_();
+
+  /**
+   * @brief Check if all rewards votes are unique
+   *
+   */
+  void checkRewardVotes();
```

Comment:
we already have checkRewardVotes in vote manager, maybe we could rename this to checkRewardVotesDuplicates or rename the other one to distinguish between them ?